helpers.go

```diff
@@ -0,0 +1,11 @@
+package fatturapa
+
+import "github.com/invopop/gobl/num"
+
+func formatPercentage(p *num.Percentage) string {
+	if p == nil {
+		return ""
+	}
+
+	return p.Amount.Multiply(*num.NewAmount(100, 0)).Rescale(2).String()
```

Comment:
@samlown Surely there's gotta be a cleaner way to do this... Ideally I want to rescale a percentage and call `StringWithoutSymbol()`. But right now `Rescale()` returns a `num.Amount`